syfertext/span.py

```diff
@@ -21,14 +21,7 @@ class Span(AbstractObject):
     """
 
     def __init__(
-        self,
-        doc: "Doc",
-        start: int,
-        end: int,
-        id: int = None,
-        owner: BaseWorker = None,
-        tags: List[str] = None,
-        description: str = None,
+        self, doc: "Doc", start: int, end: int, id: int = None, owner: BaseWorker = None,
```

Comment:
Please put one argument per line for better readability

Reply:
again `black` doesn't allow 😞

Reply:
ok then :/